requirements/dev.txt

```diff
@@ -604,14 +604,15 @@ tornado==6.4
 typing-extensions==4.11.0
     # via
     #   -r requirements/ci.txt
+    #   black
     #   zgw-consumers
 uritemplate==4.1.1
     # via
     #   -r requirements/ci.txt
     #   coreapi
     #   drf-spectacular
     #   drf-yasg
-urllib3==1.26.15
```

Comment:
@annashamray do you remember why `urllib3` was pinned to `1.26.15` in https://github.com/maykinmedia/objecttypes-api/commit/170dd771d568aad07442f01bbc311793cbbf92a7? Is it still relevant?

Reply:
I think I had problems with OpenSSL and urllib3 2.0, and some github issue advised me to downgrade urllib3 and it fixed the problem. If you don't have import errors with openssl, then it should be fine